### PR TITLE
REL-847161 Update documentation considering changes for DataGrid integration

### PR DESCRIPTION
**Pull request by joannapolrolniczakrelativity**
* Add missing settings to GitHub
* Modify user samples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1129,4 +1131,49 @@ List of samples:
...
 
+<br><br>
+
+---
+# Performance Best Practices
+
+## Import Job Settings
+
+### Encoding
+For improved performance when dealing with fileshare data on ADLS, we highly recommend encoding files in UTF-16. By doing so, you can avoid the need for conversion to the correct encoding, leading to significant time savings in your document and image workflows.
```
Comment (dawidstoga-rel) [inline README.md:1142]: Maybe is worth to mention that this refers to extracted text or other long text files. (not load file)
"...we highly recommend using extracted text or other long text files encoded in UTF-16"

File: Samples/.Net7ClientConsole/SampleCollection/Sample05_ImportDocumentsWithExtractedText.cs [hunk 2]
```diff
@@ -41,6 +41,7 @@ public async Task Sample05_ImportDocumentsWithExtractedText()
 			const int controlNumberColumnIndex = 0;
 			const int extractedTextPathColumnIndex = 12;
 			const int fileNameColumnIndex = 13;
+			const int fileSizeColumnIndex = 14;
```
Comment (dawidstoga-rel) [inline Samples/.Net7ClientConsole/SampleCollection/Sample05_ImportDocumentsWithExtractedText.cs:44]: Do our sample loadfiles (.dat) contain such a column already?

File: README.md (hunk 1 above)
Comment (joannapolrolniczakrelativity) [reply README.md]: done

File: README.md [hunk 3]
```diff
@@ -1129,4 +1131,49 @@ List of samples:
...
+					.WithAutoNumberImages()
+					.WithoutProduction()
+					.WithExtractedText(e => e.WithEncoding("UTF-16"))
+					.WithFileTypeAutoDetection())
+				.WithoutFieldsMapped()
+				.WithoutFolders();
+ 
+
+### FileSizeColumnIndex
+Another valuable setting that can enhance performance is the **FieldMapping.FileSizeColumnIndex**. By configuring this setting, the need for manual file size calculations can be eliminated. The file sizes will be automatically extracted from the load file, streamlining the process and saving valuable processing time.
```
Comment (sylwesterpe) [inline README.md:1174]: What about changing "manual" to "additional"?

**Review by dawidstoga-rel — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (joannapolrolniczakrelativity) [reply README.md]: good idea, done

**Review by mateuszKulikRelativity — APPROVED** [review]

**Pull request merged**